src/Styleguide/Pages/Artist/__stories__/CurrentEvent.story.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { Section } from "../../../Utils/Section"
+import { CurrentEvent, LargeCurrentEvent } from "../Components/CurrentEvent"
+
+storiesOf("Styleguide/Artist", module).add("Current Event", () => {
+  return (
+    <React.Fragment>
+      <Section title="Responsive">
```

Comment:
I'd started naming these just `Responsive` to begin with, but I think we should fully name them. `Responsive Current Event` or w/e. That way if you're taking screen shots or gifs or something it captures the full context.

Reply:
That makes sense, will update